app/views/admin/people/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<center><h1 class="droidFont">Editing Person</h1></center>
```

Comment:
The droidFont class is meant to be temporary. Usually, there would be a global CSS style for all HTML pages to use for <p> and <H1> etc, but I am not sure how it would affect Novi and Jeannette's pages. So I opt to just deploy the font style as a separate class for my pages first. I figured we could hash out the CSS later as a team in person.

What do you think? If you would like me to define the CSS via the body, I can do it.